src/components/Form.js

```diff
@@ -172,8 +172,19 @@ class Form extends React.Component {
     validate(values) {
         FormActions.setErrors(this.props.formID, null);
         FormActions.setErrorFields(this.props.formID, null);
+
+        // Run any validations passed as a prop
         const validationErrors = this.props.validate(values);
 
+        // Validate the input for html tags. It should superseed any other error
+        _.every(values, (inputValue, inputID) => {
+            if (!inputValue || inputValue.search(/<(.|\n)*?>/g) === -1) {
+                return;
+            }
+
+            validationErrors[inputID] = this.props.translate('common.error.inputHasHtml');
```

Comment:
Let's rename this to `common.error.invalidCharacter` so that it matches the text